lib/web/Cargo.toml

```diff
@@ -42,10 +42,13 @@ crate-type = ["cdylib"]
 wasm-opt = false
 
 [features]
-default = ["generate", "issue", "verify"]
+default = ["generate", "issue", "verify", "invoke", "delegate"]
 
 generate = []
 issue = []
 verify = []
 credential = []
 presentation = []
+delegate = []
```

Comment:
Does it make sense to support delegations but not invocations (or vise-versa)? (I'm not sure about the current setup of feature flags in general)

Reply:
it might make sense for different use cases (e.g. a kepler node really only needs to verify both but not issue, a kepler client only really needs to know how to issue invocations, a delegator client only needs to know how to issue delegations etc.) but imo most of the time it will all be supported, as duplicating the wasm is far heavier than packaging everything together. I just tried to follow the existing VC feature conventions.